<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui_Preferences</class>
 <widget class="QDialog" name="gui_Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="ctrl_actionButtons">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QListWidget" name="ctrl_preferenceTabList">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <property name="text">
         <string>General Settings</string>
        </property>
        <property name="icon">
         <iconset resource="gui_resources.qrc">
          <normaloff>:/Images/pref_gears.png</normaloff>:/Images/pref_gears.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Channel Settings</string>
        </property>
        <property name="icon">
         <iconset resource="gui_resources.qrc">
          <normaloff>:/Images/pref_sliders.png</normaloff>:/Images/pref_sliders.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Appearance</string>
        </property>
        <property name="icon">
         <iconset resource="gui_resources.qrc">
          <normaloff>:/Images/pref_appearance.png</normaloff>:/Images/pref_appearance.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QStackedWidget" name="ctrl_preferenceTabs">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page_generalSettings">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="10" column="0" colspan="2">
          <widget class="QLabel" name="lbl_startupProfile">
           <property name="text">
            <string>Startup Profile</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="ctrl_showTooltipOnMinimize">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="11" column="0" colspan="2">
          <widget class="QLabel" name="lbl_shutdownProfile">
           <property name="text">
            <string>Shutdown Profile</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="ctrl_minimizeToTray">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="ctrl_minimizeOnStart">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="6">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl_ShowTooltipOnMinimize">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Show Tooltips while Minimized</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="ctrl_quitOnClose">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_MinimizeToTray">
           <property name="text">
            <string>Minimize to Tray</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_minimizeOnStart">
           <property name="text">
            <string>Start Minimized</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_quitOnClose">
           <property name="text">
            <string>Quit on Close</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2" colspan="4">
          <widget class="QComboBox" name="ctrl_startupProfile">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="11" column="2" colspan="4">
          <widget class="QComboBox" name="ctrl_shutdownProfile">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QCheckBox" name="ctrl_alwaysShowTrayIcon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3" colspan="2">
          <widget class="QLabel" name="lbl_alwaysShowTrayIcon">
           <property name="text">
            <string>Always Show Tray Icon</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_channelSettings">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0" colspan="4">
          <widget class="QLabel" name="lbl_useLogRpmScale">
           <property name="text">
            <string>RPM sliders use logarithmic scale</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="ctrl_useLogRpmScale">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="6">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" rowspan="2" colspan="6">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item row="3" column="1">
             <widget class="QLineEdit" name="ctrl_channel3FanName"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_channel2Name">
              <property name="text">
               <string>Channel 2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="ctrl_channel1FanName"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="ctrl_channel5FanName"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="ctrl_channel2FanName"/>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="ctrl_channel4FanName"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lbl_channel3Name">
              <property name="text">
               <string>Channel 3</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lbl_channel4Name">
              <property name="text">
               <string>Channel 4</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="lbl_channel5Name">
              <property name="text">
               <string>Channel 5</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbl_channel1Name">
              <property name="text">
               <string>Channel 1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="ctrl_channel1TempName"/>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="ctrl_channel2TempName"/>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="ctrl_channel3TempName"/>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="ctrl_channel4TempName"/>
            </item>
            <item row="6" column="2">
             <widget class="QLineEdit" name="ctrl_channel5TempName"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_channelNameHeader">
              <property name="text">
               <string>Fan Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="lbl_channelTempHeader">
              <property name="text">
               <string>Temperature Probe</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_appearanceSettings">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="1">
          <widget class="QComboBox" name="ctrl_style"/>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="ctrl_showChannelLabels">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_selectStyle">
           <property name="text">
            <string>Style</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_showChannelLabels">
           <property name="text">
            <string>Show Channel Labels</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ctrl_preferenceTabList</tabstop>
  <tabstop>ctrl_minimizeToTray</tabstop>
  <tabstop>ctrl_alwaysShowTrayIcon</tabstop>
  <tabstop>ctrl_minimizeOnStart</tabstop>
  <tabstop>ctrl_quitOnClose</tabstop>
  <tabstop>ctrl_showTooltipOnMinimize</tabstop>
  <tabstop>ctrl_startupProfile</tabstop>
  <tabstop>ctrl_shutdownProfile</tabstop>
  <tabstop>ctrl_channel1FanName</tabstop>
  <tabstop>ctrl_channel1TempName</tabstop>
  <tabstop>ctrl_channel2FanName</tabstop>
  <tabstop>ctrl_channel2TempName</tabstop>
  <tabstop>ctrl_channel3FanName</tabstop>
  <tabstop>ctrl_channel3TempName</tabstop>
  <tabstop>ctrl_channel4FanName</tabstop>
  <tabstop>ctrl_channel4TempName</tabstop>
  <tabstop>ctrl_channel5FanName</tabstop>
  <tabstop>ctrl_channel5TempName</tabstop>
  <tabstop>ctrl_useLogRpmScale</tabstop>
  <tabstop>ctrl_actionButtons</tabstop>
 </tabstops>
 <resources>
  <include location="gui_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ctrl_actionButtons</sender>
   <signal>accepted()</signal>
   <receiver>gui_Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctrl_actionButtons</sender>
   <signal>rejected()</signal>
   <receiver>gui_Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
